.editorconfig

```diff
@@ -0,0 +1,21 @@
+root = true
```

Comment:
We're trying to use the mega linter together with the editorconfig-checker, right? If so, then I think this configuration is not what we need based on the [documentation](https://editorconfig-checker.github.io) 🤔

Reply:
Can you point out to what exactly you're referring to and quote that please?

Or just feel free to experiment and change in your own if you see potential improvements, maybe that's more efficient.

Reply:
> Configuration
> 
> There are a bunch of command line flags you can use, but there is also a configuration file which you can use in  your current working directory called .ecrc. If you want to use a different config you can pass the -config <path>- flag to the binary. You can generate a configuration with the init -flag. A sample configuration file can look like this:
> 
> {
>   "Verbose": false,
>   "Debug": false,
>   "IgnoreDefaults": false,
>   "SpacesAftertabs": false,
>   "NoColor": false,
>   "Exclude": [],
>   "AllowedContentTypes": [],
>   "PassedFiles": [],
>   "Disable": {
>     "EndOfLine": false,
>     "Indentation": false,
>     "InsertFinalNewline": false,
>     "TrimTrailingWhitespace": false
>   }
> }

Sorry, this is what I'm talking about 🤗 I can of course experiment with this 👍🏼

Reply:
As I understand it this is an alternative to the configuration via `.editorconfig`. As said, the `.editorconfig` is taken into account, not sure if you've seen it. I have the feeling that we're on quite different pages somehow...

Reply:
Ahh 💡, after a deeper look into the configuration and the logs of the GitHub Actions, I did understand that it is doing what it has to do 👍🏼 

For me this is a good start, let's merge it 🤗 Thanks for implementing this @magro!